packages/p2p/src/components/modal-manager/modals/rate-change-modal/rate-change-modal.jsx

```diff
@@ -12,22 +11,13 @@ const RateChangeModal = ({ currency }) => {
         client: { local_currency_config },
     } = useStore();
 
-    const { floating_rate_store } = useStores();
     const local_currency = currency ?? local_currency_config?.currency;
     const { hideModal, is_modal_open } = useModalManagerContext();
 
-    const closeModal = () => {
-        floating_rate_store.setIsMarketRateChanged(false);
-        hideModal();
-    };
+    const closeModal = () => hideModal();
 
     return (
-        <Modal
-            is_open={is_modal_open && floating_rate_store.is_market_rate_changed}
-            toggleModal={closeModal}
-            small
-            className='rate-changed-modal'
-        >
+        <Modal is_open={is_modal_open} toggleModal={closeModal} small className='rate-changed-modal'>
```

Comment:
`suggestion`: could we pass hideModal here instead now since we've removed the other function inside closeModal?

```suggestion
        <Modal is_open={is_modal_open} toggleModal={hideModal} small className='rate-changed-modal'>
```